tetris.py

```diff
@@ -475,16 +563,22 @@ def flasher(self, full_lines, fill):
             for x_line in full_lines:
                 self.main_board.draw_rectangle(x, x_line, "flash", fill)
 
+    def get_time(self):
+        game_time = time.time() - self.start_time - self.paused_time
+        return f"{int(game_time / 60):02d}:{int(game_time % 60):02d}"
+
     def timer(self):
         if self.status == GameStatus.in_progress:
-            game_time = time.time() - self.start_time - self.paused_time
-            self.topbar_time.config(text=f"{int(game_time / 60):02d}:{int(game_time % 60):02d}")
+            self.topbar_time.config(text=self.get_time())
             self.topbar_time.after(1000, self.timer)
 
     def game_over(self):
         y_coordinates = [y for (x, y) in self.coord_extractor()]
         if any(y < 0 for y in y_coordinates):
             self.status = GameStatus.game_over
+            json_dict['high_scores'].append({'Time': self.get_time(), 'Game Speed': game_speed, 'Score': self.score})
```

Comment:
The JSON contains times as strings, making it difficult to sort the contents of the treeview, in case we want to do that later.